tests/test_prep_6_10_upgrade.py

```diff
@@ -0,0 +1,31 @@
+from testfm.log import logger
+
+
+def test_positive_prep_6_10_upgrade(ansible_module):
+    """Runs content directory preparation for pulp3 migration.
+
+    :id: 46531c4a-2276-4ccd-aed7-92c53d1ad267
+
+    :setup:
+        1. foreman-maintain should be installed.
+
+    :steps:
+        1. Create /var/lib/pulp/content directory with root owner and 0000 mode
+        2. Run foreman-maintain prep-6.10-upgrade
+
+    :expectedresults: Owner group and mode should be changed.
+
+    :CaseImportance: High
+    """
+    content_dir = "/var/lib/pulp/content"
+    ansible_module.file(path=content_dir, state="directory", owner="root", mode="0000")
```

Comment:
Trying to understand the rationale behind this, why we are keeping mode="0000"? 

And I guess the default owner/Logged-In user will be root only for satellite, could you check that? if the default owner is a root, so no need to pass separately.

Reply:
The reason is that the `prep-6.10-upgrade` command shall update the mode with SGID and g+rwx only and should not touch anything else (_user_ and _all_ permissions), which we are ensuring by the 0000 preset.

Yes, the default is _root_ and could be omitted. Passing it explicitly is just for better readability - look here the owner is root and after the `prep-6.10-upgrade` command it is _pulp_, which is exactly the expected behaviour. Although, it should have been `group="root"` instead of `owner="root"`. I would update it with group unless you strongly insist on omitting.

Reply:
Okay gotcha, if this works for you, I won't insist to change.